smarty/templates/login.tpl

```diff
@@ -28,8 +28,8 @@
             </div>
           </form>
           <div class="help-links">
-            <a href="{$baseurl}/password-reset">Forgot your password?</a><br/>
-            <a href="{$baseurl}/request-account">Request Account</a>
+            <a href="{$baseurl}/password-reset/">Forgot your password?</a><br/>
+            <a href="{$baseurl}/request-account/">Request Account</a>
```

Comment:
this is necessary since the htacess is expecting that slash (see link below)

https://github.com/aces/Loris/blob/minor/htdocs/.htaccess#L9